### ISSUE TITLE
Cargo test fails 

### ISSUE THREAD

**Issue by pm100** [rhysd/tui-textarea#35]
It tries to compile src/lib.rs, which I includes the readme that has invalid code marked as rust.

I looked at ci.yaml and that seems to not exclude it, but it gets excluded anyway. I don't understand why. But really plain n 'cargo test' should work.

**Comment by rhysd** [comment]
It should be fixed at #34. Please update your local `main` branch to the latest then try:

```
cargo test --features search
```

Note that tests for `search` feature is not run by `cargo test` since the feature is disabled by default

**Comment by rhysd** [comment]
> I don't understand why.

Please read `README.md` as text. The trick is using `rust,ignore` code block instead of `rust`.

````markdown
...

```rust,ignore
fn main() {
  // ...
}
```
````

`ignore` marks that the code block should be skipped by doc test.
